lib/collections.js

```diff
@@ -0,0 +1,6 @@
+
+Games = new Mongo.Collection("games");
```

Comment:
Maybe add a comment here saying that its built in.  Will that change when we have users coming from facebook, google, etc?